### PR TITLE
Sawa 200 blob pictures

### PR DESCRIPTION
**Pull request by logoffro1**
Use POSTMAN to test

### CODE REVIEW

**Review by egecnr — APPROVED** [review]

File: ShowerShow/Control/BlobStorageController.cs [hunk 1]
```diff
@@ -0,0 +1,261 @@
...
+        {
+            _logger = log;
+            account = CloudStorageAccount.Parse(connection);
+        }
+
+
+        [Function("UploadProfilePicture")] // USE POSTMAN TO TEST
+        [OpenApiParameter(name: "UserId", In = ParameterLocation.Path, Required = true, Type = typeof(Guid), Description = "The User ID parameter")]
+        [OpenApiOperation(operationId: "UploadProfilePicture", tags: new[] { "BlobStorage" })]
+        public async Task<HttpResponseData> UploadProfilePicture([HttpTrigger(AuthorizationLevel.Anonymous, "post", Route = "user/{UserId:Guid}/profile/uploadpic")] HttpRequestData req, Guid UserId)
```
Comment (egecnr) [inline ShowerShow/Control/BlobStorageController.cs:64]: Dont forget to add auth bro

**Review by egecnr — CHANGES_REQUESTED** [review]

File: ShowerShow/Control/BlobStorageController.cs [hunk 2]
```diff
@@ -0,0 +1,261 @@
...
+        [OpenApiResponseWithBody(statusCode: HttpStatusCode.OK, contentType: "image/jpeg", bodyType: typeof(byte[]), Description = "The OK response with the profile picture.")]
+        public async Task<HttpResponseData> GetProfilePictureOfUser([HttpTrigger(AuthorizationLevel.Anonymous, "get", Route = "user/{UserId:Guid}/profile/getpic")] HttpRequestData req, Guid UserId)
+        {
+            _logger.LogInformation("Uploading to blob.");
+
+            string containerName = Environment.GetEnvironmentVariable("ContainerProfilePictures");
+            HttpResponseData responseData = req.CreateResponse();
+            try
+            {
+                CloudBlobClient client = account.CreateCloudBlobClient();
```
Comment (egecnr) [inline ShowerShow/Control/BlobStorageController.cs:128]: Should you check whether the user exist or not? I'm not sure about the requirement of it

**Comment by logoffro1** [comment]
I added AUTH and checking for user, also added a DELETE endpoint and a default profile pic, in case the user doesn't have one

**Comment by egecnr** [comment]
> I added AUTH and checking for user, also added a DELETE endpoint and a default profile pic, in case the user doesn't have one

GJ bro lets get this branch into dev

**Review by egecnr — APPROVED** [review]

**Pull request merged**